2021-04-28 16:26:22 [scrapy] DEBUG: Redirecting (302) to <GET https://thatsthem.com/challenge?r=%2Femail%2Flebron%40gmail.com> from <GET https://thatsthem.com/email/lebron@gmail.com>
2021-04-28 16:26:25 [scrapy] DEBUG: Crawled (200) <GET https://thatsthem.com/challenge?r=%2Femail%2Flebron%40gmail.com> (referer: None)
2021-04-28 16:26:25 [scrapy] ERROR: Spider error processing <GET https://thatsthem.com/challenge?r=%2Femail%2Flebron%40gmail.com> (referer: None)
Traceback (most recent call last):
  File "c:\users\michael\desktop\piiportal\pii-portal\venv\lib\site-packages\scrapy\utils\defer.py", line 120, in iter_errback
    yield next(it)
  File "c:\users\michael\desktop\piiportal\pii-portal\venv\lib\site-packages\scrapy\utils\python.py", line 353, in __next__
    return next(self.data)
  File "c:\users\michael\desktop\piiportal\pii-portal\venv\lib\site-packages\scrapy\utils\python.py", line 353, in __next__
    return next(self.data)
  File "c:\users\michael\desktop\piiportal\pii-portal\venv\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "c:\users\michael\desktop\piiportal\pii-portal\venv\lib\site-packages\scrapy\spidermiddlewares\offsite.py", line 29, in process_spider_output
    for x in result:
  File "c:\users\michael\desktop\piiportal\pii-portal\venv\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "c:\users\michael\desktop\piiportal\pii-portal\venv\lib\site-packages\scrapy\spidermiddlewares\referer.py", line 340, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "c:\users\michael\desktop\piiportal\pii-portal\venv\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "c:\users\michael\desktop\piiportal\pii-portal\venv\lib\site-packages\scrapy\spidermiddlewares\urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "c:\users\michael\desktop\piiportal\pii-portal\venv\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "c:\users\michael\desktop\piiportal\pii-portal\venv\lib\site-packages\scrapy\spidermiddlewares\depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "c:\users\michael\desktop\piiportal\pii-portal\venv\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "C:\Users\Michael\Desktop\PIIPortal\PII-Portal\crawler\crawler\spiders\mydomain.py", line 34, in parse
    if 'We did not find any results' in header:
TypeError: argument of type 'NoneType' is not iterable
2021-04-28 16:26:25 [scrapy] INFO: Closing spider (finished)
2021-04-28 16:26:25 [scrapy] INFO: Dumping Scrapy stats:
{'downloader/request_bytes': 639,
 'downloader/request_count': 2,
 'downloader/request_method_count/GET': 2,
 'downloader/response_bytes': 4083,
 'downloader/response_count': 2,
 'downloader/response_status_count/200': 1,
 'downloader/response_status_count/302': 1,
 'elapsed_time_seconds': 3.820453,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2021, 4, 28, 23, 26, 25, 686450),
 'log_count/DEBUG': 2,
 'log_count/ERROR': 1,
 'log_count/INFO': 7,
 'response_received_count': 1,
 'scheduler/dequeued': 2,
 'scheduler/dequeued/memory': 2,
 'scheduler/enqueued': 2,
 'scheduler/enqueued/memory': 2,
 'spider_exceptions/TypeError': 1,
 'start_time': datetime.datetime(2021, 4, 28, 23, 26, 21, 865997)}
2021-04-28 16:26:25 [scrapy] INFO: Spider closed (finished)
